index.html

```diff
@@ -204,26 +219,36 @@ <h1>About me</h1>
             </div>
         </section>
         <section class="comments">
-            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-            <form class="form_comment" action="https://formspree.io/f/xdovrwnp" method="post">
-                <input type="text" id="fname" name="fname" placeholder="Full Name" minlength="4" maxlength="30" class="input_place input_comment" required>
-                <hr class="InputSeparation">
-                <br>
-                <br>
-                <br>
-                <input id="faddress" name="faddress" class="input_place input_comment" placeholder="Email Address" type="email" minlength="10" maxlength="30" required>
-                <hr class="InputSeparation">
-                <textarea name="comment" id="comment" cols="30" rows="10" class="text_comment" type="text" minlength="7" maxlength="500" placeholder="Write some text here" required>Write me something...</textarea>
-                <button type="submit" class="msjButton">Get in touch</button>
-            </form>
+            <div class="coment_container">
+                <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+                <form class="form_comment" action="https://formspree.io/f/xdovrwnp" method="post">
+                    <div class="names_container">
+                        <div class="names_container_input">
+                            <input type="text" id="fname" name="fname" placeholder="Full Name" minlength="4" maxlength="30" class="input_place input_comment first_name" required>
+                            <hr class="InputSeparation first_name">
+                        </div>
+                        <div class="names_container_input">
+                            <input type="text" id="fsurname" name="fsurname" placeholder="Last Name" minlength="4" maxlength="30" class="input_place input_comment last_name" required>
+                            <hr class="InputSeparation last_name">
+                        </div>
+                    </div>
```

Comment:
- [ ] I know we have been asking to follow the Figma template, here comes an exception. According to the requirements, we should only have a single name input for both mobile and desktop versions regardless of the Figma template. Kindly remove the last name input and make sure the full name input occupies the whole width. 🧐️
- [ ] Please also make sure all the left side of your inputs is aligned. 🤓️

**Project requirements**
![Screenshot from 2023-06-09 07-24-34](https://github.com/alexansaa/Personal_Portafolio/assets/17243325/c065b327-c7fa-4b17-bb5d-bb0662ef3fcd)

**Your design**
![Screenshot from 2023-06-09 07-12-36](https://github.com/alexansaa/Personal_Portafolio/assets/17243325/abea4d1c-07df-472a-a78e-3bae976e54e8)